docs/podman-system.1.md

```diff
@@ -13,7 +13,7 @@ The system command allows you to manage the podman systems
 
 | Command  | Man Page                                            | Description                                                                  |
 | -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
-| info     | [podman-system-info(1)](podman-info.1.md)           | Displays Podman related system information.                                  |
+| info     | [podman-info(1)](podman-info.1.md)                  | Displays Podman related system information.                                  |
```

Comment:
This is wrong there is a podman-system-info.

Reply:
There is a `podman system info` command; there is no such man page:
```
$ ll docs/*info*
-rw-rw-r--. 1 esm esm 3933 Feb 28 07:23 docs/podman-info.1
-rw-rw-r--. 1 esm esm 3775 Feb 13 14:18 docs/podman-info.1.md
```
(IOW: this is the `podman-system` man page, and it correctly lists an `info` subcommand, but it is actually documented in `podman-info(1)`. Without my PR, a user would incorrectly believe there is a `podman-system-info(1)` page.)

Reply:
There is a man page, 

find . -name podman-system-info\*
./docs/links/podman-system-info.1
cat docs/links/podman-system-info.1 
.so man1/podman-info.1

Reply:
Install  the package and run man page and it will work.

Reply:
Oh.... this changes everything. Thank you.